pkg/domain/filters/volumes.go

```diff
@@ -34,15 +35,15 @@ func GenerateVolumeFilters(filtersValues url.Values) ([]libpod.VolumeFilter, err
 					return v.Scope() == scopeVal
 				})
 			case "label":
-				filter := val
 				vf = append(vf, func(v *libpod.Volume) bool {
-					return filters.MatchLabelFilters([]string{filter}, v.Labels())
+					return filters.MatchLabelFilters(values, v.Labels())
```

Comment:
I think this is a bit ugly and not consistent with the containers, pods filter functions.
We should not loop over values and instead pass the values array to each individual filter function and loop inside there.